tests/test_annotations.py

```diff
@@ -26,33 +26,34 @@
 saved_urls = {}
 
 
-def setup_module():
-    destination = pathlib.Path("_test_cache/zipfiles/")
+def setUpModule():
```

Comment:
The conventions for this in pytest are `setup_module()`/`teardown_module()`. https://docs.pytest.org/en/latest/how-to/xunit_setup.html?highlight=setup_module